app/components/agenda/agenda-header.hbs

```diff
@@ -224,17 +228,11 @@
 {{/if}}
 
 {{#if this.isEditingSession}}
-  <Auk::Modal @size="large">
-    <Auk::Modal::Header
-      @title={{t "edit-session"}}
-      @onClose={{this.toggleEditingSession}}
-    />
-    <Sessions::Forms::EditSession
-      @meeting={{@meeting}}
-      @successfullyEdited={{this.toggleEditingSession}}
-      @cancelForm={{this.toggleEditingSession}}
-    />
-  </Auk::Modal>
+  <Meeting::EditMeetingModal
+    @meeting={{@meeting}}
+    @didSave={{this.toggleEditingSession}}
+    @onCancel={{this.toggleEditingSession}}
```

Comment:
Ik weet dat het wat buiten de scope van deze story valt, maar voor de duidelijkheid zou ik in `Agenda::AgendaHeader` alles wat met meeting te maken `meeting` noemen ipv `session` (`isEditingSession` -> `isEditingMeeting`; `toggleEditingSession` -> `toggleEditingMeeting`)